Makefile

```
@@ -157,6 +165,16 @@ tests: test-md  test-sh  test-py  test-ccpp
 	@echo "All tests & checks have been completed successfully."
 	@echo ""
 
+# former scripts/ci/buildAll.sh
+build-all:
+	@mkdir  -p  $(OUT_DIR)
+	@chmod  0777  $(OUT_DIR)
+	cd  source  &&  bash  ./build.sh
+	@echo "All Firmware built"
+	@cp  -r  $(OUT_HEX)/*.bin  $(OUT_DIR)
+	@cp  -r  $(OUT_HEX)/*.hex  $(OUT_DIR)

```

```suggestion
	@cp  -r  $(OUT_HEX)/*.hex  $(OUT_DIR)
	@cp  -r  $(OUT_HEX)/*.def  $(OUT_DIR)

```

Best we grab dfu files as well.

---

Haven't thought about that. Sure!